src/Dns/Cname/CnameRecord.php

```diff
@@ -65,7 +65,7 @@ public function update(UpdateRequest $request)
                 'host'              => (string) $request->record(),
                 'current-value'     => (string) $request->currentValue(),
                 'new-value'         => (string) $request->newValue(),
-                'ttl'               => $request->ttl(),
+                'ttl'               => (int) $request->ttl()->integer(),
```

Comment:
No need to cast this as you're casting it to an int in the method.

Reply:
Don't _need_ it for any of them, strings as casted in their respective methods as well